$NetBSD: patch-retrie.cabal,v 1.2 2023/01/28 02:40:22 pho Exp $

Although we want the executable "retrie" to be built, we don't want
"demo-retrie". The upstream should really decouple these.

--- retrie.cabal.orig	2001-09-09 01:46:40.000000000 +0000
+++ retrie.cabal
@@ -118,10 +118,7 @@ executable retrie
   default-language: Haskell2010
 
 executable demo-retrie
-  if flag(BuildExecutable)
-    Buildable: True
-  else
-    Buildable: False
+  Buildable: False
   main-is:
     Main.hs
   hs-source-dirs: demo hse
